ios/engine/KMEI/KeymanEngine/Classes/LanguagePicker/LanguageLMDetailViewController.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  LanguageLMDetailViewController.swift
+//  KeymanEngine
```

Comment:
- [ ] Minor issue - this view doesn't always seem to be accessible for models.  The (i) info button isn't presently displaying for the English - MTNT model within the lexical model picker.